naming/src/main/java/com/alibaba/nacos/naming/core/v2/service/impl/PersistentClientOperationServiceImpl.java

```diff
@@ -170,6 +195,12 @@ public Response onApply(WriteRequest request) {
                 case DELETE:
                     onInstanceDeregister(instanceRequest.service, instanceRequest.getClientId());
                     break;
+                case CHANGE:
+                    if (instanceAndServiceExist(instanceRequest)) {
```

Comment:
Discussion: would be better an atomic operation?

Reply:
> Discussion: would be better an atomic operation?

The "log" of "jraft" is executed in sequence, and the next log will not be executed before it is executed, so I don't think it's a big problem here.